gulpfile.js

```diff
@@ -144,10 +146,26 @@ const javascripts = {
   }
 };
 
-const hashJavascriptFilenames = () => src(paths.dist + 'javascripts/*.js')
+const hashJavascriptFilenames = () => src([
+  paths.dist + 'javascripts/*.js',
+  paths.dist + 'javascripts/*.js.map'
+])
   .pipe(plugins.hash())
   .pipe(dest(paths.dist + 'javascripts/'));
 
+const fixSourcemapFilenames = () => src(paths.dist + 'javascripts/*.js-*.map')
```

Comment:
Rather than use a plugin and have to hack around it, could we just write own our JS instead? That should be no longer than the length of this.